website/docs/r/vdc.html.markdown

```diff
@@ -0,0 +1,124 @@
+* `vm_quota` - (Optional) The maximum number of VMs that can be created in this VDC. Includes deployed and undeployed VMs in vApps and vApp templates. Defaults to 0, which specifies an unlimited number.
+* `is_enabled` - (Optional) True if this VDC is enabled for use by the organization VDCs. A VDC is always enabled on creation.
+* `storage_profile` - (Required) Storage profiles supported by this VDC.  See [Storage Profile](#storageprofile) below for details.
+* `resource_guaranteed_memory` - (Optional) Percentage of allocated memory resources guaranteed to vApps deployed in this VDC. For example, if this value is 0.75, then 75% of allocated resources are guaranteed. Required when AllocationModel is AllocationVApp or AllocationPool. Value defaults to 1.0 if the element is empty.
+* `resource_guaranteed_cpu` - (Optional) Percentage of allocated CPU resources guaranteed to vApps deployed in this VDC. For example, if this value is 0.75, then 75% of allocated resources are guaranteed. Required when AllocationModel is AllocationVApp or AllocationPool. Value defaults to 1.0 if the element is empty.
+* `v_cpu_in_mhz` - (Optional) Specifies the clock frequency, in Megahertz, for any virtual CPU that is allocated to a VM. A VM with 2 vCPUs will consume twice as much of this value. Ignored for ReservationPool. Required when AllocationModel is AllocationVApp or AllocationPool, and may not be less than 256 MHz. Defaults to 1000 MHz if the element is empty or missing.
+* `is_thin_provision` - (Optional) Boolean to request thin provisioning. Request will be honored only if the underlying datastore supports it. Thin provisioning saves storage space by committing it on demand. This allows over-allocation of storage.
+* `network_pool_name` - (Optional) Reference to a network pool in the Provider VDC. Required if this VDC will contain routed or isolated networks.
+* `provider_vdc_name` - (Required) A name of the Provider VDC from which this organization VDC is provisioned.
+* `uses_fast_provisioning` - (Optional) Boolean to request fast provisioning. Request will be honored only if the underlying datastore supports it. Fast provisioning can reduce the time it takes to create virtual machines by using vSphere linked clones. If you disable fast provisioning, all provisioning operations will result in full clones.
+* `over_commit_allowed` - (Optional) Set to false to disallow creation of the VDC if the AllocationModel is AllocationPool or ReservationPool and the ComputeCapacity you specified is greater than what the backing Provider VDC can supply. Defaults to true if empty or missing.
+* `vm_discovery_enabled` - (Optional) True if discovery of vCenter VMs is enabled for resource pools backing this VDC. If left unspecified, the actual behaviour depends on enablement at the organization level and at the system level.
```

Comment:
What about some boolean params to begin with "enable"? Here would be `enable_vm_discovery`.

BTW, which GUI element represents this field?

Reply:
Changed. Seems this property isn't in GUI

Reply:
Please also add default value description - from current one it's not clear.

Reply:
There isn't default. If provide true means one thing if provide false means another thing. If you don't configure means `behaviour depends on enablement at the organization level and at the system level.`

Reply:
Sorry, but still not clear. What does happen if you define it to false?

Reply:
Got it, please expand the description to reflect all three possible states so it's really clear for the users, something like this:
"If true, discovery of vCenter VMs is enabled for <...>. If false, discovery is disabled. If left unspecified, <...>."

Reply:
Changed